src/app/pages/DashboardPage/LearningMaterials.tsx

```diff
@@ -32,30 +33,71 @@ const StyledLink = styled(Link)(() => ({
   },
 }))
 
-const getContent = (t: TFunction, layer: Layer) => {
-  switch (layer) {
-    case Layer.emerald:
-      return {
-        description: t('learningMaterials.emerald.description'),
-        header: t('learningMaterials.emerald.header'),
-        link: docs.emerald,
-        title: t('common.emerald'),
-      }
+type LayerContent = {
+  description: string
+  header: string
+  url: string
+}
+type NetworkContent = Partial<Record<Layer, LayerContent[]>>
```

Comment:
Personally I would go with some kind of object, instead of array. This implies that it is possible to have 0,1,2, ... N possible combinations. Can we al least make it `[LayerContent, LayerContent, LayerContent]` type.

Reply:
OK, switching to an object.